### PR TITLE
Release the Ceph Metricbeat module as GA

### PR DESCRIPTION
**Pull request by ruflin**
* Add docs
* Fix default config
* Extend reference config

### CODE REVIEW

File: metricbeat/module/ceph/_meta/config.yml [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
   #username: "user"
   #password: "secret"
 
 - module: ceph
-  #metricsets:
-  #  - cluster_disk
-  #  - osd_tree
-  #  - osd_df
-  #  - pool_disk
+  metricsets:
```
Comment (ruflin) [inline metricbeat/module/ceph/_meta/config.yml]: I think this was actually a "bug" before as it meant the module is started twice with all default metricsets. Should we backport this fix to 6.3?

**Comment by herver** [comment]
Hi there,

Sorry to jump on this PR, but we're using Ceph here and it turns out that the ceph-rest-api is deprecated since the Luminous release and has been removed in Mimic (see http://docs.ceph.com/docs/mimic/releases/luminous/)

It has been replaced by a ceph-mgr plugin (http://docs.ceph.com/docs/luminous/mgr/restful/#).

**Comment by ruflin** [comment]
@herver Thank you for the heads up. Will need to investigate more here.

Let's hold off merging this PR for now.

**Comment by ruflin** [comment]
I'm closing this PR and created a follow up issue here to track the implementation of the new API here: https://github.com/elastic/beats/issues/7723

@herver Appreciate a lot that you jumped in here and provided the above details.

**Pull request closed without merging**